pdx-extract/app/config.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+
+def map_level(level):
+    """ Maps logging level strings to logging level codes
+
+    Parameters:
+        level (String): The level string to be mapped.
+
+    Returns:
+        Integer: Number that matches the logging level.
+    """
+    return {"critical": 50, "error": 40, "warning": 30, "info": 20, "debug": 10}.get(
+        level, 10
+    )
+
+
+BUCKET_NAME = os.environ.get("BUCKET_NAME", "my-different-bucket")
```

Comment:
Is this a sensible default? Consider: If only some of these variables are specified, will the program still behave usefully / in a way you can reason about based on the other defaults? I don't know for your business case, but generally, I don't see pushing data to a general bucket as a useful default -- better just to give some invalid configuration error.

Reply:
It's really not, it doesn't represent the type of data that is going to be stored in that bucket.

So it seems like you're recommending having a default value of `None` or something similar, and when accessing the `BUCKET_NAME` check if it is invalid. Otherwise, if we attempt to get an environment variable that is undefined it will throw an exception.

Reply:
Right! Nothing wrong with having default environment variables, just easy to be too generous with defaults to the point that the behavior isn't _useful_ even if it operates (especially when you get into a situation where it looks like you're writing correctly but it's not showing up in your bucket -- and then realize it's writing to the default bucket)